ccl/backupccl: TestBackupTenantImportingTable failed

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9599352?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9599352?buildTab=artifacts#/) on master @ [8e6cdbd8b3d12ba5d65db101a3abfeb2cb1bf470](https://github.com/cockroachdb/cockroach/commits/8e6cdbd8b3d12ba5d65db101a3abfeb2cb1bf470):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable515955645
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 856487830635249665: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable515955645
--- FAIL: TestBackupTenantImportingTable (54.82s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/disaster-recovery
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-26985

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9616513?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9616513?buildTab=artifacts#/) on master @ [51fcf796c5595549e90fca3cc1fd8165ee14fd6c](https://github.com/cockroachdb/cockroach/commits/51fcf796c5595549e90fca3cc1fd8165ee14fd6c):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable815391443
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 856773103113175041: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable815391443
--- FAIL: TestBackupTenantImportingTable (55.35s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9627454?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9627454?buildTab=artifacts#/) on master @ [c08946af1f935d543a033de6b7b585c4d65288c4](https://github.com/cockroachdb/cockroach/commits/c08946af1f935d543a033de6b7b585c4d65288c4):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable3678705847
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 857057384872542209: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable3678705847
--- FAIL: TestBackupTenantImportingTable (55.27s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9638820?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9638820?buildTab=artifacts#/) on master @ [3382b2737ceb72de9b14d1d5923773718be5bca7](https://github.com/cockroachdb/cockroach/commits/3382b2737ceb72de9b14d1d5923773718be5bca7):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable2681345774
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 857338802692063233: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable2681345774
--- FAIL: TestBackupTenantImportingTable (55.16s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9658840?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9658840?buildTab=artifacts#/) on master @ [1cd507a7c6582fd91bb56123e62bd4fde45c4d22](https://github.com/cockroachdb/cockroach/commits/1cd507a7c6582fd91bb56123e62bd4fde45c4d22):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable3165262740
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 857623481868943361: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable3165262740
--- FAIL: TestBackupTenantImportingTable (55.37s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9680619?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9680619?buildTab=artifacts#/) on master @ [a4ab6b6243f509e62993b4d81855f23470d9901a](https://github.com/cockroachdb/cockroach/commits/a4ab6b6243f509e62993b4d81855f23470d9901a):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable728246317
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 857904056696963073: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable728246317
--- FAIL: TestBackupTenantImportingTable (55.03s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9699832?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9699832?buildTab=artifacts#/) on master @ [b767225a6893ea6e81d50618d634dbba92d3b680](https://github.com/cockroachdb/cockroach/commits/b767225a6893ea6e81d50618d634dbba92d3b680):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable3857053077
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 858189157067390977: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable3857053077
--- FAIL: TestBackupTenantImportingTable (55.25s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9738475?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9738475?buildTab=artifacts#/) on master @ [10665f9856d9c7ca51557bdf5b48fe9df796228b](https://github.com/cockroachdb/cockroach/commits/10665f9856d9c7ca51557bdf5b48fe9df796228b):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable3061912150
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 858754579202998273: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable3061912150
--- FAIL: TestBackupTenantImportingTable (55.01s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9750452?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9750452?buildTab=artifacts#/) on master @ [23ea65e4b5cfaa5e02298cddfbe8f9fbe581456b](https://github.com/cockroachdb/cockroach/commits/23ea65e4b5cfaa5e02298cddfbe8f9fbe581456b):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable4291241863
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 859041954257043457: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable4291241863
--- FAIL: TestBackupTenantImportingTable (55.23s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9782270?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9782270?buildTab=artifacts#/) on master @ [847de4595b62a0c03b14a3d0865cc8222fcf0f61](https://github.com/cockroachdb/cockroach/commits/847de4595b62a0c03b14a3d0865cc8222fcf0f61):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable2869937859
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 859602873522356225: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable2869937859
--- FAIL: TestBackupTenantImportingTable (55.38s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9804204?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9804204?buildTab=artifacts#/) on master @ [2f4231084e27760b4927e15da9b6d46705dc8552](https://github.com/cockroachdb/cockroach/commits/2f4231084e27760b4927e15da9b6d46705dc8552):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable543470356
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 859884989114187777: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable543470356
--- FAIL: TestBackupTenantImportingTable (54.53s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsGcpLinuxX8664BigVm_CclUnitTests/9811445?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsGcpLinuxX8664BigVm_CclUnitTests/9811445?buildTab=artifacts#/) on master @ [2f4231084e27760b4927e15da9b6d46705dc8552](https://github.com/cockroachdb/cockroach/commits/2f4231084e27760b4927e15da9b6d46705dc8552):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/08eca5aa191e7cb5498d48e4d60049c7/logTestBackupTenantImportingTable1159847306
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 859896331265835009: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/08eca5aa191e7cb5498d48e4d60049c7/logTestBackupTenantImportingTable1159847306
--- FAIL: TestBackupTenantImportingTable (55.42s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #102311 ccl/backupccl: TestBackupTenantImportingTable failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsGcpLinuxX8664BigVm_CclUnitTests/9811445?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsGcpLinuxX8664BigVm_CclUnitTests/9811445?buildTab=artifacts#/) on master @ [2f4231084e27760b4927e15da9b6d46705dc8552](https://github.com/cockroachdb/cockroach/commits/2f4231084e27760b4927e15da9b6d46705dc8552):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/08eca5aa191e7cb5498d48e4d60049c7/logTestBackupTenantImportingTable166643779
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 859896541785751553: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/08eca5aa191e7cb5498d48e4d60049c7/logTestBackupTenantImportingTable166643779
--- FAIL: TestBackupTenantImportingTable (54.82s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #102311 ccl/backupccl: TestBackupTenantImportingTable failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsGcpLinuxX8664BigVm_CclUnitTests/9811445?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsGcpLinuxX8664BigVm_CclUnitTests/9811445?buildTab=artifacts#/) on master @ [2f4231084e27760b4927e15da9b6d46705dc8552](https://github.com/cockroachdb/cockroach/commits/2f4231084e27760b4927e15da9b6d46705dc8552):


```
=== RUN   TestBackupTenantImportingTable
I230426 06:20:57.656636 1 (gostd) rand.go:243  [-] 1  random seed: 8043429034968481275
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/08eca5aa191e7cb5498d48e4d60049c7/logTestBackupTenantImportingTable3331870941
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 859896754007670785: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/08eca5aa191e7cb5498d48e4d60049c7/logTestBackupTenantImportingTable3331870941
--- FAIL: TestBackupTenantImportingTable (54.88s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #102311 ccl/backupccl: TestBackupTenantImportingTable failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupTenantImportingTable [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9849075?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9849075?buildTab=artifacts#/) on master @ [3b1bc872bb4bf8e5eb2ad8790ba8ec781167dd1f](https://github.com/cockroachdb/cockroach/commits/3b1bc872bb4bf8e5eb2ad8790ba8ec781167dd1f):


```
=== RUN   TestBackupTenantImportingTable
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable2490199193
    test_log_scope.go:79: use -show-logs to present logs inline
    backup_tenant_test.go:101: condition failed to evaluate within 45s: pq: job 860456626416877569: job with status reverting cannot be requested to be canceled
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/95e138e66d69292427dfb9528cf06d04/logTestBackupTenantImportingTable2490199193
--- FAIL: TestBackupTenantImportingTable (55.33s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #102315 ccl/backupccl: TestBackupTenantImportingTable failed [C-test-failure O-robot T-disaster-recovery branch-release-22.2]
- #102313 ccl/backupccl: TestBackupTenantImportingTable failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #102311 ccl/backupccl: TestBackupTenantImportingTable failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupTenantImportingTable.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/backupccl/key_rewriter.go
+++ b/pkg/ccl/backupccl/key_rewriter.go
@@ -229,6 +229,19 @@ func (kr *KeyRewriter) RewriteKey(key []byte, wallTime int64) ([]byte, bool, err
 	// in which case we are restoring as a system tenant.
 	if kr.fromSystemTenant && bytes.HasPrefix(key, keys.TenantPrefix) {
 		k, ok := kr.tenants.rewriteKey(key)
+		if ok {
+			// Skip keys from ephemeral cluster status tables so that the restored
+			// cluster does not observe stale leases/liveness until it expires.
+			noTenantPrefix, _, err := keys.DecodeTenantPrefix(key)
+			if err != nil {
+				return nil, false, err
+			}
+			_, tableID, _ := keys.SystemSQLCodec.DecodeTablePrefix(noTenantPrefix)
+
+			if tableID == keys.SQLInstancesTableID || tableID == keys.SqllivenessID || tableID == keys.LeaseTableID {
+				return k, false, nil
+			}
+		}
 		return k, ok, nil
 	}
 
@@ -281,6 +294,12 @@ func (kr *KeyRewriter) checkAndRewriteTableKey(key []byte, wallTime int64) ([]by
 	// they will be caught later on if tableID isn't in descs or kr doesn't
 	// perform a rewrite.
 	_, tableID, _ := keys.SystemSQLCodec.DecodeTablePrefix(key)
+
+	// Skip keys from ephemeral cluster status tables so that the restored cluster
+	// does not observe stale leases/liveness until it expires.
+	if tableID == keys.SQLInstancesTableID || tableID == keys.SqllivenessID || tableID == keys.LeaseTableID {
+		return key, false, nil
+	}
 
 	desc := kr.descs[descpb.ID(tableID)]
 	if desc == nil {

--- a/pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go
+++ b/pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go
@@ -298,7 +298,7 @@ func ingestWithRetries(
 		if jobs.IsPermanentJobError(err) || errors.Is(err, context.Canceled) {
 			break
 		}
-		const msgFmt = "stream ingestion waits for retrying after error %s"
+		const msgFmt = "stream ingestion waits for retrying after error: %q"
 		log.Warningf(ctx, msgFmt, err)
 		updateRunningStatus(ctx, execCtx, ingestionJob, jobspb.ReplicationError,
 			fmt.Sprintf(msgFmt, err))

--- a/pkg/ccl/streamingccl/streamingest/stream_ingestion_processor.go
+++ b/pkg/ccl/streamingccl/streamingest/stream_ingestion_processor.go
@@ -654,15 +654,8 @@ func (sip *streamIngestionProcessor) consumeEvents() (*jobspb.ResolvedSpans, err
 	return nil, nil
 }
 
-func (sip *streamIngestionProcessor) rekey(key roachpb.Key) ([]byte, error) {
-	rekey, ok, err := sip.rekeyer.RewriteKey(key, 0 /*wallTime*/)
-	if !ok {
-		return nil, errors.New("every key is expected to match tenant prefix")
-	}
-	if err != nil {
-		return nil, err
-	}
-	return rekey, nil
+func (sip *streamIngestionProcessor) rekey(key roachpb.Key) ([]byte, bool, error) {
+	return sip.rekeyer.RewriteKey(key, 0 /*wallTime*/)
 }
 
 func (sip *streamIngestionProcessor) bufferSST(sst *kvpb.RangeFeedSSTable) error {
@@ -714,13 +707,20 @@ func (sip *streamIngestionProcessor) bufferRangeKeyVal(
 	defer sp.Finish()
 
 	var err error
-	rangeKeyVal.RangeKey.StartKey, err = sip.rekey(rangeKeyVal.RangeKey.StartKey)
+	var ok bool
+	rangeKeyVal.RangeKey.StartKey, ok, err = sip.rekey(rangeKeyVal.RangeKey.StartKey)
 	if err != nil {
 		return err
 	}
-	rangeKeyVal.RangeKey.EndKey, err = sip.rekey(rangeKeyVal.RangeKey.EndKey)
+	if !ok {
+		return nil
+	}
+	rangeKeyVal.RangeKey.EndKey, ok, err = sip.rekey(rangeKeyVal.RangeKey.EndKey)
 	if err != nil {
 		return err
+	}
+	if !ok {
+		return nil
 	}
 	sip.rangeBatcher.buffer(rangeKeyVal)
 	return nil
@@ -749,9 +749,13 @@ func (sip *streamIngestionProcessor) bufferKV(kv *roachpb.KeyValue) error {
 	}
 
 	var err error
-	kv.Key, err = sip.rekey(kv.Key)
+	var ok bool
+	kv.Key, ok, err = sip.rekey(kv.Key)
 	if err != nil {
 		return err
+	}
+	if !ok {
+		return nil
 	}
 
 	if sip.rewriteToDiffKey {

--- a/pkg/storage/fingerprint_writer.go
+++ b/pkg/storage/fingerprint_writer.go
@@ -116,8 +116,12 @@ func (f *fingerprintWriter) PutRawMVCCRangeKey(key MVCCRangeKey, bytes []byte) e
 func (f *fingerprintWriter) PutRawMVCC(key MVCCKey, value []byte) error {
 	defer f.hasher.Reset()
 	// Hash the key/timestamp and value of the RawMVCC.
-	if err := f.hashKey(key.Key); err != nil {
+	err, skip := f.hashKey(key.Key)
+	if err != nil {
 		return err
+	}
+	if skip {
+		return nil
 	}
 	if err := f.hashTimestamp(key.Timestamp); err != nil {
 		return err
@@ -134,9 +138,14 @@ func (f *fingerprintWriter) PutUnversioned(key roachpb.Key, value []byte) error 
 	defer f.hasher.Reset()
 
 	// Hash the key and value in the absence of a timestamp.
-	if err := f.hashKey(key); err != nil {
+	err, skip := f.hashKey(key)
+	if err != nil {
 		return err
 	}
+	if skip {
+		return nil
+	}
+
 	if err := f.hashValue(value); err != nil {
 		return err
 	}
@@ -145,14 +154,27 @@ func (f *fingerprintWriter) PutUnversioned(key roachpb.Key, value []byte) error 
 	return nil
 }
 
-func (f *fingerprintWriter) hashKey(key []byte) error {
+func (f *fingerprintWriter) hashKey(key []byte) (error, bool) {
+	noTenantPrefix, err := keys.StripTenantPrefix(key)
+	if err != nil {
+		return err, false
+	}
+	// Fingerprinting ignores rows from a few special-cased key ranges, namely for
+	// the tables that contain ephemeral cluster-topology/state information, which
+	// if expected to differ between two clusters that otherwise contain the same
+	// data.
+	_, tID, _, _ := keys.DecodeTableIDIndexID(noTenantPrefix)
+	if tID == keys.SqllivenessID || tID == keys.LeaseTableID || tID == keys.SQLInstancesTableID {
+		return nil, true
+	}
+
 	if f.options.StripIndexPrefixAndTimestamp {
-		return f.hash(f.stripIndexPrefix(key))
+		return f.hash(f.stripIndexPrefix(key)), false
 	}
 	if f.options.StripTenantPrefix {
-		return f.hash(f.stripTenantPrefix(key))
+		return f.hash(noTenantPrefix), false
 	}
-	return f.hash(key)
+	return f.hash(key), false
 }
 
 func (f *fingerprintWriter) hashTimestamp(timestamp hlc.Timestamp) error {
@@ -187,14 +209,6 @@ func (f *fingerprintWriter) stripValueChecksum(value []byte) []byte {
 		return value
 	}
 	return value[mvccChecksumSize:]
-}
-
-func (f *fingerprintWriter) stripTenantPrefix(key []byte) []byte {
-	remainder, err := keys.StripTenantPrefix(key)
-	if err != nil {
-		return key
-	}
-	return remainder
 }
 
 func (f *fingerprintWriter) stripIndexPrefix(key []byte) []byte {
@@ -261,11 +275,19 @@ func FingerprintRangekeys(
 	defer fw.Close()
 	fingerprintRangeKey := func(stack MVCCRangeKeyStack) (uint64, error) {
 		defer fw.hasher.Reset()
-		if err := fw.hashKey(stack.Bounds.Key); err != nil {
+		err, skip := fw.hashKey(stack.Bounds.Key)
+		if err != nil {
 			return 0, err
 		}
-		if err := fw.hashKey(stack.Bounds.EndKey); err != nil {
+		if skip {
+			return 0, nil
+		}
+		err, skip = fw.hashKey(stack.Bounds.EndKey)
+		if err != nil {
 			return 0, err
+		}
+		if skip {
+			return 0, nil
 		}
 		for _, v := range stack.Versions {
 			if err := fw.hashTimestamp(v.Timestamp); err != nil {

